pkgs/by-name/fc/fcast-receiver/package.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitLab
+, makeDesktopItem
+, copyDesktopItems
+, makeWrapper
+, electron
+}:
+
+buildNpmPackage rec {
+  pname = "fcast-receiver";
+  version = "1.0.12";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.futo.org";
+    owner = "videostreaming";
+    repo = "fcast";
+    rev = "b13d0f7e8150c279d377a78f89d338b7fc0f5539";
+    hash = "sha256-NqOQFEP5MWFNWHj6ZpNCVh8D/4YlNhqA/Db6hdAXepo=";
+  };
+
+  sourceRoot = "${src.name}/receivers/electron";
+
+  makeCacheWritable = true;
+  dontPatchELF = true;
```

Comment:
Why not?

Reply:
I've cheked, and removed dontPatchELF as it was not needed, probably remained in from testing. makeCacheWritable was needed however, package failed to build without that